### PR TITLE
Copyedit phase 2

### LINKED ISSUE #6323: typo in tutorial command
File: [docker-for-windows/index.md](https://docs.docker.com/docker-for-windows/), CC @gbarr01

The example command to stop webserver docker in the tutorial has a typo.

docker container stpp webserver should be 
docker container stop webserver

obvious for sure ;)

### PR DESCRIPTION
**Pull request by gbarr01**
- Added text on PowerShell
- Added @thaJeztah 's compressed images, plus some
- Removed unnecessary images
- More copy editing 

Fixes #6323 

cc: @akimd

### CODE REVIEW

File: docker-for-windows/index.md [hunk 1]
```diff
@@ -13,23 +13,26 @@ redirect_from:
...
+Docker for Windows is the best way to get started with Docker _on Windows_.
 
 > See [Install Docker for Windows](install.md){: target="_blank" class="_"} for information on system requirements and stable & edge channels.
 
 ## Test your installation
 
-1.  Open "Command Prompt (`cmd`) and run `docker --version ` to ensure that you have a supported version of Docker:
+1.  Open a terminal window (Command Prompt or PowerShell, _but not_ PowerShell ISE).
+
+2.  Run `docker --version ` to ensure that you have a supported version of Docker:
```
Comment (akimd) [inline docker-for-windows/index.md]: spurious whitespace

File: docker-for-windows/index.md [hunk 2]
```diff
@@ -99,31 +103,38 @@ by running something more complex, an OS and a webserver.
...
 2.  You are in the container. At the root `#` prompt, check the `hostname` of the container:
 
     ```shell
     root@8aea0acb7423:/# hostname
     8aea0acb7423
     ```
-    The hostname is assigned the container ID (and matches the ID in the prompt).
+    Notice that the hostname is assigned as the container ID (and is also used in the prompt).
+
+3.  Exit and stop the container with the `exit` command:
```
Comment (akimd) [inline docker-for-windows/index.md]: This is slightly misleading: exit quits the shell, so the container stops.

**Comment by akimd** [comment]
Impressive job!

File: docker-for-windows/index.md (hunk 1 above)
Comment (gbarr01) [reply docker-for-windows/index.md]: If you mean the 2 spaces after the number, not spurious. It's 2 "atom" tabs which makes indenting code easier.

Comment (akimd) [reply docker-for-windows/index.md]: I mean "`docker --version `".

**Pull request merged**